src/components/dataDisplays/IndividualsDisplay.jsx

```diff
@@ -75,6 +75,7 @@ export default function IndividualsDisplay({
       data={individuals}
       title={title}
       loading={loading}
+      maxHeight={600}
```

Comment:
That's a good point. Are we convinced that every instantiation of IndividualsDisplay is going to be paginated?

Reply:
`IndividualsDisplay` passes its `rest` props to its `DataDisplay` component, so if an instance would rather have a maximum height than be paginated, it can pass `maxHeight`.

Reply:
Ok, I've noticed two weird things now:
1. When passing maxHeight to IndividualsDisplay, the DataDisplay does NOT seem to be respecting it:
<IndividualsDisplay
        individuals={searchResults || []}
        hideFilterSearch
        loading={loading}
        sortExternally
        searchParams={searchParams}
        setSearchParams={setSearchParams}
        dataCount={resultCount}
        style={{ maxHeight: 600 }}
      />

2. The limit: 20 param in useFilterIndividuals.js also seems not to be respected. I created an investigation ticket for this latter one: DEX-1322.

Reply:
Removed in https://github.com/WildMeOrg/codex-frontend/pull/430/commits/c1cb541a950de76a1c8d3f3c560526c479976212. So, currently that means that this table is very large.

Reply:
For the first weird thing: You are not technically passing `maxHeight` to `IndividualsDisplay`, you are passing `style`. Neither of those two props are used directly by `IndividualsDisplay`. Instead they are destructured into its `rest` property and spread as props to `DataDisplay`. `DataDisplay` does destructure  `maxHeight`, but `style` is once again destructured into the `rest` property. The `maxHeight` prop is used in the `TableContainer`'s `style`, but `rest` is spread to the `div` wrapping all of `DataDisplay`'s other children. That `div` has the default `overflow: visible`, so it may not look like `maxHeight` is working, but it is.

When I pass `style={{ maxHeight: 200, border: '1px solid red' }}` to `IndividualsDisplay`, it is only 200px tall, but it overflows. Notice that the pagination is right below the 200px tall display:
<img width="673" alt="max-height-wrapper" src="https://user-images.githubusercontent.com/50299119/180572199-5ce1c274-9042-443a-b707-736f252696ae.png">

When I pass `maxHeight={200}` to `IndividualsDisplay`, the table height is constrained:
<img width="673" alt="max-height-table" src="https://user-images.githubusercontent.com/50299119/180572207-fc007ca1-27bd-4263-82c5-c30552655db2.png">

Reply:
Maybe instead of adding a `maxHeight` prop to `DataDisplay`, we should add a `tableContainerStyles` prop instead?